index.js

```diff
@@ -12,3 +12,177 @@ document.querySelector('.popup-menu .menu-item').addEventListener('click', close
 document.querySelector('.popup-menu .menu-item2').addEventListener('click', closePopUp);
 document.querySelector('.popup-menu .menu-item3').addEventListener('click', closePopUp);
 document.querySelector('.popup-nav').addEventListener('click', closePopUp);
+document.querySelector('.popup-nav');
+
+// Project Cards
+
+const ProjecData = [
```

Comment:
[OPTIONAL] Hey 👋🏽 for nomenclature reasons and best practices, I suggest you use `camelcase` for Variable name but it should start with a lowercase letter, it's better to use capital case letters for `Classes` 🤓